charts/pathfinder-rpc-server/README.md

```diff
@@ -2,35 +2,46 @@
 |--------------------------------------------------------------------------|-----------------------|------------------------------------------------------------------------------------------|
 | [blockchain-indexer](https://github.com/CirclesUBI/blockchain-indexer)   | blockchain-indexer    | Indexes circles related on-chain events and writes them to a postgres db                 |
-| [pathfinder2](https://github.com/CirclesUBI/pathfinder2)                 | pathfinder2           | Finds transitive payment paths between untrusted users                                   | 
-| [pathfinder2-updater](https://github.com/CirclesUBI/pathfinder2-updater) | pathfinder2-updater   | Updates the 'pathfinder2' with data from the 'blockchain-indexer'                        | 
+| [pathfinder2](https://github.com/CirclesUBI/pathfinder2)                 | pathfinder2           | Finds transitive payment paths between untrusted users                                   |
+| [pathfinder2-updater](https://github.com/CirclesUBI/pathfinder2-updater) | pathfinder2-updater   | Updates the 'pathfinder2' with data from the 'blockchain-indexer'                        |
 | [pathfinder-proxy](https://github.com/CirclesUBI/pathfinder-proxy)       | pathfinder-proxy      | Maintains statistics, load balances and filters requests to the 'pathfinder2'            |
 
 
+#### Notes
+Since the installation of pathfinder and updater are in the same container, this charts needs to explicitely set the ports for both in the service so these can be located by other services such as the pathfinder-proxy on the network.
+
+For an in detail explanation of the every component and variable please refer to the repos above.
+
 ### Pre-requisities
 
 Create the following secrets in your cluster:
     - blockchain-indexer-connection-string: connection string for the indexer to the db
     - pathfinder-updater-connection-string: connection string for the pathfinder to the db
-    - indexer-db-secret: `password` for the `doadmin` user and `postgres-password`  for the `postgres` user 
+    - indexer-db-secret: `password` for the `doadmin` user and `postgres-password`  for the `postgres` user
+
+Manage your secrets with your prefered tool.
 
-Manage your secrets with your prefered tool. 
 
-### Currents caveats and usage 
+### Currents caveats and usage
 
 #### Caveats
-- The indexer db is empty and without proper schemas and there is not an indexer-db-init. It only exists for local development with ganache. 
-- To syncronise the indexer fully at the start it requires a lot resources and its own rpc node. There is still the option to install your own rpc node if you want to go down this road and avoid the following workaround. However, for quicker sync and  resources saving  we recommend the workaround. 
+- The indexer db is empty and without proper schemas, indexes and reference since there is not an indexer-db-init. It only exists for local development with ganache.
```

Comment:
```suggestion
- The indexer database is empty and without proper schemas, indexes and reference since there is no indexer-db-init. It only exists for local development with ganache.
```